contracts/utils/addresses.js

```diff
@@ -30,6 +30,8 @@ addresses.mainnet.DAI = "0x6B175474E89094C44Da98b954EedeAC495271d0F";
 addresses.mainnet.USDC = "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48";
 addresses.mainnet.USDT = "0xdAC17F958D2ee523a2206206994597C13D831ec7";
 addresses.mainnet.TUSD = "0x0000000000085d4780B73119b644AE5ecd22b376";
+addresses.mainnet.USDS = "0xdc035d45d973e3ec169d2276ddab16f1e407384f";
+addresses.mainnet.sUSDS = "0xa3931d71877c0e7a3148cb7eb4463524fec27fbd";
```

Comment:
What do you think of using checksum address instead?
- `0xdC035D45d973E3EC169d2276DDab16f1e407384F`
- `0xa3931d71877C0E7a3148CB7Eb4463524FEc27fbD`